bricks/presentational-bricks/src/markdown-display/SyntaxHighlighter.module.css

```
@@ -0,0 +1,180 @@
+html[data-theme="dark-v2"] {
+  .codeBlockWrapper:hover .copyButton:hover {
+    opacity: 1;
+    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.15) !important;
+    border: 1px solid #dadde1;
+  }
+
+  .copyButton {
+    border-color: #606770 !important;
+    color: #d9d9d9 !important;
+    color: #979595;
+
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)